go.mod

```diff
@@ -3,19 +3,17 @@ module github.com/CrunchyData/crunchy-bridge-operator
 go 1.16
 
 require (
-	github.com/RHEcosystemAppEng/dbaas-operator v0.1.3
+	github.com/RHEcosystemAppEng/dbaas-operator v0.1.5-0.20220329142537-6708678c607a
 	github.com/go-logr/logr v0.4.0
 	github.com/google/uuid v1.1.2
 	github.com/jpillora/backoff v1.0.0
 	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.13.0
-	k8s.io/api v0.21.1
-	k8s.io/apiextensions-apiserver v0.21.1
-	k8s.io/apimachinery v0.21.1
-	k8s.io/client-go v0.21.1
-	k8s.io/utils v0.0.0-20210709001253-0e1f9d693477
-	sigs.k8s.io/controller-runtime v0.9.0
+	github.com/onsi/gomega v1.15.0
+	k8s.io/api v0.22.1
+	k8s.io/apiextensions-apiserver v0.22.1
+	k8s.io/apimachinery v0.22.1
+	k8s.io/client-go v0.22.1
+	k8s.io/utils v0.0.0-20210802155522-efc7438f0176
+	sigs.k8s.io/controller-runtime v0.10.0
```

Comment:
why are you updating the k8s modules? was it required for newcache?

Reply:
this is auto-generated

Reply:
hmm... please revert to former versions and only update dbaas-operator module, unless these changes are required.

Reply:
seeing cockroachdb version,  kind of smiliar version
``` require (
	github.com/RHEcosystemAppEng/dbaas-operator v0.1.3
	github.com/go-logr/logr v0.4.0
	github.com/onsi/ginkgo v1.16.4
	github.com/onsi/gomega v1.15.0
	github.com/parnurzeal/gorequest v0.2.16
	k8s.io/api v0.22.1
	k8s.io/apimachinery v0.22.1
	k8s.io/client-go v0.22.1
	k8s.io/utils v0.0.0-20210802155522-efc7438f0176
	moul.io/http2curl v1.0.0 // indirect
	sigs.k8s.io/controller-runtime v0.10.0
)```

is this due to the SKD version? I am using operator-sdk version: "v1.15.0"

Reply:
hmm, try 1.10.1

Reply:
same with version SDK 1.10.1. the dependencies generated automatically, keep as it.

Reply:
yea, i think that has more to do the the `newcache` code changes... i seem to remember this requiring a k8s module updates